Makefile.am

```diff
@@ -2892,11 +2892,14 @@ apidoc: py-apidoc
 endif
 
 .PHONY: py-apidoc
-py-apidoc: epydoc.conf $(RUN_IN_TEMPDIR) $(GENERATED_FILES)
+py-apidoc: $(RUN_IN_TEMPDIR) $(GENERATED_FILES)
 	env - PATH="$$PATH" PYTHONPATH="$$PYTHONPATH" \
-	$(RUN_IN_TEMPDIR) epydoc -v \
-	  --conf $(CURDIR)/epydoc.conf \
-	  --output $(CURDIR)/$(APIDOC_PY_DIR)
+	$(RUN_IN_TEMPDIR) pydoctor -v \
+	  --config $(CURDIR)/pydoctor.ini \
+	  --html-output $(CURDIR)/$(APIDOC_PY_DIR) \
+          $(lint_python_code)
+
+
 
```

Comment:
Thanks a lot @newellz2.

Do the above two new lines look to much?

However, please also cleanup the `Makefile.am` with occurrence of epydoc: `epydoc.conf` target itself and from `CLEANFILES`, `epydoc.conf.in` from `EXTRA_DIST` (add `pydoctor.ini` therefore).

Reply:
@saschalucas It is too much. 😀 

I cleaned up the references to epydoc in Makefile.am and the py-apidoc target.